tests/vmi_multus_test.go

```diff
@@ -766,5 +769,101 @@ func pingVirtualMachine(vmi *v1.VirtualMachineInstance, ipAddr, prompt string) {
 		&expect.BSnd{S: "echo $?\n"},
 		&expect.BExp{R: "0"},
 	}, 30)
-	Expect(err).ToNot(HaveOccurred())
+	Expect(err).ToNot(HaveOccurred(), "Failed to ping VMI %s within the given timeout", vmi.Name)
+}
+
+// Tests in Multus suite are expecting a Linux bridge to be available on each node, with iptables allowing
+// traffic to go through. This function creates a Daemon Set on the cluster (if not exists yet), this Daemon
+// Set creates a linux bridge and configures the firewall. We use iptables-compat in order to work with
+// both iptables and newer nftables.
+// TODO: Once kubernetes-nmstate is ready, we should use it instead
+func configureNodeNetwork(virtClient kubecli.KubevirtClient) {
+	// Privileged DaemonSet configuring host networking as needed
+	networkConfigDaemonSet := appsv1.DaemonSet{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "apps/v1",
+			Kind:       "DaemonSet",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "network-config",
+			Namespace: "kube-system",
```

Comment:
metav1 has constant for system namespace, please replace all `kube-system` with it

Reply:
Done